cmd/resource-topology-exporter/main.go

```diff
@@ -219,27 +220,33 @@ func argsParse(argv []string) (nrtupdater.Args, resourcemonitor.Args, resourceto
 		}
 		resourcemonitorArgs.ExcludeList.ExcludeList = conf.ExcludeList
 		localArgs.SysConf = conf.Resources
+		// do not overwrite with empty an existing value (e.g. from opts)
+		if rteArgs.TopologyManagerPolicy == "" {
+			rteArgs.TopologyManagerPolicy = conf.TopologyManagerPolicy
+		}
 	}
+
 	if tmPolicy, ok := arguments["--topology-manager-policy"].(string); ok {
-		if tmPolicy == "" {
-			// last attempt
-			tmPolicy = os.Getenv("TOPOLOGY_MANAGER_POLICY")
+		// empty string is a valid value here, so just keep going;
+		// but not overwrite with empty an existing value (e.g. from conf)
+		if rteArgs.TopologyManagerPolicy == "" {
```

Comment:
I think that explicitly mentioning the topology-manager-policy value should be precedence over what is written in the config which is more general and includes other stuff as well.